packages/workspace/src/schematics/move/lib/update-jest-config.ts

```diff
@@ -57,7 +57,7 @@ export function updateJestConfig(schema: Schema): Rule {
 
         const newRootJestConfigContent = oldRootJestConfigContent.replace(
           findProject,
-          `<rootDir>/${destination}`
+          `'<rootDir>/${destination}'`
```

Comment:
You could probably change the regex on line 54 to be this:
```
const { libsDir, appsDir } = getWorkspaceLayout(tree);
const findProject = new RegExp(
  `<rootDir>\/(${libsDir}|${appsDir})\/${schema.projectName}`,
  'g'
);
```

> The `getWorkspaceLayout` is imported from `./utils`. This is important for users who changed their default layout. 

If you remove the quotes from the regex, you'll still be able to find the string if users are using `'` or `"`. Then you don't have to run a format, because the string between the quotes will be replaced. 

Preferably, using AST would've been better, but the [jest config update function](https://github.com/nrwl/nx/blob/3ea76b78dec98e6758343063ebc9bff43a7fe5ac/packages/jest/src/migrations/utils/config/legacy/update-config.ts#L4) is bound to the `@nrwl/jest` package and we can't import that here easily.

Reply:
For the time being, I can change the implementation tonight like you mentioned